### ISSUE TITLE
python 2 to python 3

### ISSUE THREAD

**Issue by wt123u** [freesunshine0316/nary-grn#10]
Hi @freesunshine0316 ,
Thanks for sharing! 
Would you consider upgrading your code to python 3? When I executed the code in different versions (python 2 and python 3), the results were quite different, as follows:
python 2: Validation Data Eval:
                Dev loss = 27.9529,  Dev accu = 0.5500 110/200
python 3: Validation Data Eval:
               Dev loss = 17.8237,   Dev accu = 0.8250 165/200

**Comment by freesunshine0316** [comment]
Hi,
Thanks for pointing it out, it could be some problem related to python version, as someone else reported a similar issue. I'm currently busy on something else, I'll try to do it but the timeline can't be guaranteed.

**Comment by leslie991123** [comment]
嗨，可以分享一下您的python3的代码吗，谢谢
